packages/cfa-template/template/backend/serverless.yml

```diff
@@ -10,6 +10,7 @@ provider:
     environment:
```

Comment:
Also need to add https://github.com/lefthookhq/42matters--frigg/blob/7da7c8f6c26a7cf151530449acbf2073da5ce974/backend/serverless.yml#L10 (lines 10-14)